pyproject.toml

```diff
@@ -153,7 +151,7 @@ exclude = ["**.py"]
 "notebooks/tabular_examples/tree_based_models/tree_shap_paper/*"  = ["ALL"]
 
 # Disable some unwanted rules on Jupyter notebooks
-"*.ipynb" = ["E703"]  # Trailing semicolons, used to supress outputs
+"*.ipynb" = ["E703", "E402"]  # Allow trailing semicolons, allow imports not at top
```

Comment:
Good question. Some of the notebooks have some imports in context under the relevant subheading, rather than at the top. For example [here](https://shap.readthedocs.io/en/latest/example_notebooks/tabular_examples/model_agnostic/Iris%20classification%20with%20scikit-learn.html). there are a few subheadings with examples of explaining different types of models, and each model is imported in the relevant section.

I think this is fine - in documentation, sometimes things are clearer this way, when one particular import might be relevant only for one section.